modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -8,11 +8,41 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
   PATH_ENABLED_FOR_ENV = Settings.modules_appeals_api.documentation.notice_of_disagreements_v1
 
   NOD_TAG = ['Notice of Disagreements'].freeze
+  OBJ = :object
 
   read_file = ->(path) { File.read(AppealsApi::Engine.root.join(*path)) }
   read_json = ->(path) { JSON.parse(read_file.call(path)) }
   read_json_from_same_dir = ->(filename) { read_json.call(['app', 'swagger', 'appeals_api', 'v1', filename]) }
 
+  response_nod_show_not_found = read_json_from_same_dir['response_nod_show_not_found.json']
+  example_all_fields_used = read_json[['spec', 'fixtures', 'valid_10182.json']]
+
+  response_nod_show_success = lambda do
+    properties = {
+      status: { '$ref': '#/components/schemas/nodStatus' },
```

Comment:
These look like Open API refs to the JSON schema. Is that reference being populated using the swagger-blocks DSL?